<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndividuGraphique</class>
 <widget class="QWidget" name="IndividuGraphique">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>62</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>10</y>
     <width>851</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ajoutIndividu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>246</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Ajouter une personne</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="listeIndividus"/>
    </item>
    <item>
     <widget class="QPushButton" name="modifierIndividu">
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="supprimerIndividu">
      <property name="maximumSize">
       <size>
        <width>246</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
